bindata/network/ovn-kubernetes/common/ipsec-pre414.yaml

```diff
@@ -174,6 +182,11 @@ spec:
           }
           trap cleanup SIGTERM
 
+          if rpm --dbpath=/usr/share/rpm -q libreswan; then
+            echo "host have libreswan, ovn-ipsec will sleep infinity"
```

Comment:
"host has libreswan and therefore ipsec will be configured by <other host IPSEC DS name?>, this ovn ipsec container will sleep to infinity"